dispatch/static/manager/src/js/vendor/dispatch-editor/components/Editor.js

```diff
@@ -88,13 +89,13 @@ class ContentEditor extends React.Component {
   componentWillReceiveProps(nextProps) {
     // Push content to editor state
     this.setState({
-      editorState: EditorState.push(this.state.editorState, nextProps.content)
+      editorState: EditorState.push(this.state.editorState, nextProps.content || ContentState.createFromText(''))
```

Comment:
Instead of passing in `ContentState.createFromText('')`, you should be able to wrap the whole call like this:

```js
if (nextProps.content) {
  this.setState({editorState: EditorState.push(this.state.editorState, nextProps.content)})
}
```

The component is initialized with an empty state, so there's no need to pass it in again. Give it a try, though -- I could be wrong.